toolbox/artifact_downloader.py

```diff
@@ -59,13 +69,14 @@ def latest_artifact_run() -> GitHub.Run:
   return latest_run
 
 
-if __name__ == '__main__':
+def parse_args() -> argparse.Namespace:
```

Comment:
Should also support download artifacts from a given run id. Currently the downloader downloads from the latest run.